### ISSUE TITLE
Style guide should state the "Other" pattern used with enums

### ISSUE THREAD

**Issue by cjerdonek** [votinginfoproject/vip-specification#106]
The style guide should describe the following pattern used throughout the spec of adding an "Other" element with type `xs:string` whenever we want to allow values not included in an enumeration:

```xml
<xs:element name="VoteVariationType" type="VoteVariation" minOccurs="0" />
<xs:element name="OtherVoteVariationType" type="xs:string" minOccurs="0" />
```

**Comment by cjerdonek** [comment]
Created PR #115.

**Comment by jdmgoogle** [comment]
Looks Good To Me

**Comment by cjerdonek** [comment]
Thanks. @pstenbjorn how does this look to you? @jungshadow requested sign off from two people.

**Comment by cjerdonek** [comment]
Closed by PR #115.
